README.md

```diff
@@ -103,7 +104,7 @@ Full customizable variables:
   --colorFocusInformation: #375de733;
   --colorInformationDark: #2c4ab8;
   --colorInformationDarker: #21388b;
-  --fontFamily: Roboto,'Helvetica Neue',Helvetica,Arial,sans-serif;
+  --fontFamily: Roboto, 'Helvetica Neue', helvetica, arial, sans-serif;
```

Comment:
Is this an autofix? @tinhvqbk

Reply:
It's manual, but missing some things, let's me fix it

Reply:
fixed https://github.com/gotitinc/aha-css/commit/959a0c8421ac6e39685f6ad198c32597890ac141